XDMA/linux-kernel/xdma/alinx_arch.h

```diff
@@ -110,19 +114,12 @@ struct qav_state {
 	timestamp_t available_at;
 };
 
-struct buffer_tracker {
-	sysclock_t free_at[HW_QUEUE_SIZE];
-	int head; // Insert
-	int tail; // Remove
-	int count;
-};
-
 struct tsn_config {
 	struct qbv_config qbv;
 	struct qbv_baked_config qbv_baked;
 	struct qav_state qav[VLAN_PRIO_COUNT];
 	struct mqprio_config mqprio;
-	struct buffer_tracker buffer_tracker;
+	uint32_t buffer_count;
```

Comment:
`buffer_count` is ambiguous. It could be used buffer or available buffer.
Let's change it

Reply:
I renamed it to `entry_count` and added it to `struct buffer_tracker`.

Reply:
I still curious that `entry_count` is suitable name for it.
Any idea?

Reply:
How about `queued_packets` or `queued_packet_count`?

Reply:
pending, holding...
`pending_packets`